tests/models/test_granite.py

```diff
@@ -50,7 +50,7 @@ def test_granite_3_8b_overrides():
             cache_config=NO_SWAP_CONFIG,
         )
 
-        assert granite_3_8b_config.cache_config.num_gpu_blocks_override == 2080
+        assert granite_3_8b_config.cache_config.num_gpu_blocks_override == 8192
```

Comment:
As travis mentioned, you could check the torch_sendnn version to see if it already has support 8K. The version can be found: `torch_sendnn._version.__version__`. But I think the code should use 8192 as the default and downgrade to 2080 if an old version of torch sendnn is found instead of the contrary because if someone is hacking on a local environment where they have editable installs directly from local git repos the version information might be wrong. Also `0.0.0` is the same thing as "version not found": https://github.ibm.com/ai-chip-toolchain/torch_sendnn/blob/546c73752389137f3ef18ffa7f18e99476bc87e4/torch_sendnn/_version.py#L19C17-L19C22